### ISSUE TITLE
Management of custom modules

### ISSUE THREAD

**Issue by PassiveLemon** [comfyanonymous/ComfyUI#571]
This doesn't really have to be specific to this repository but I figure it's the best place to ask. What is the future support for custom modules going to look like? Are they all going to be merged into here? I feel like a GitHub specifically for custom modules would be a good idea so they can be easily accessible without having to specifically search for them.

**Comment by WASasquatch** [comment]
Have you seen this: https://github.com/comfyanonymous/ComfyUI/discussions/470

Most these plugins are custom nodes. And having complicated github install commands to install from a unified repo when people may not want everything doesn't seem like a good way to go about it.

**Comment by PassiveLemon** [comment]
I don't mean that every single one gets installed at once but more so like what you just linked: A place where a list of plugins can be seen with links. Maybe they could be specifically added to be installed by some extension manager some point in the future but this is what I was looking for.

**Comment by WASasquatch** [comment]
> I don't mean that every single one gets installed at once but more so like what you just linked: A place where a list of plugins can be seen with links. Maybe they could be specifically added to be installed by some extension manager some point in the future but this is what I was looking for.

Yeah I was hoping @comfyanonymous would link from readme cause a lot of people are unaware of a lot of nodes for things they are requesting or looking for. And Google is no help, you have to search GitHub.

**Comment by comfyanonymous** [comment]
I added it to the examples page since I thought it would be more visible there: https://comfyanonymous.github.io/ComfyUI_examples/
